[1,"\u001B[0m\u001B[1m\u001B[31merror\u001B[0m: Uncaught (in promise) Error: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 17, columns 5--10) Field \"\u001B[34mtheme\u001B[39m\" has empty value but it must instead be a string\n16: \u001B[34m  html:\u001B[39m\n17: \u001B[34m    theme:\u001B[39m\n             ~\n18: \u001B[34m    light: flatly\u001B[39m\n\n\nIn file _quarto.yml\n(line 17, columns 5--10) Field \"\u001B[34mtheme\u001B[39m\" has empty value but it must instead be a string\n16: \u001B[34m  html:\u001B[39m\n17: \u001B[34m    theme:\u001B[39m\n             ~\n18: \u001B[34m    light: flatly\u001B[39m\n\n        throw new ValidationError2(errorMessage, yamlValidationErrors);\n\u001B[0m\u001B[31m              ^\u001B[0m\n    at \u001B[0m\u001B[1m\u001B[3mreadAndValidateYamlFromFile\u001B[0m (\u001B[0m\u001B[36mfile:///opt/quarto/bin/quarto.js\u001B[0m:\u001B[0m\u001B[33m20063\u001B[0m:\u001B[0m\u001B[33m15\u001B[0m)\n    at async \u001B[0m\u001B[36mfile:///opt/quarto/bin/quarto.js\u001B[0m:\u001B[0m\u001B[33m45521\u001B[0m:\u001B[0m\u001B[33m26\u001B[0m\n    at async \u001B[0m\u001B[1m\u001B[3mprojectContext\u001B[0m (\u001B[0m\u001B[36mfile:///opt/quarto/bin/quarto.js\u001B[0m:\u001B[0m\u001B[33m45376\u001B[0m:\u001B[0m\u001B[33m26\u001B[0m)\n    at async \u001B[0m\u001B[1m\u001B[3mrender\u001B[0m (\u001B[0m\u001B[36mfile:///opt/quarto/bin/quarto.js\u001B[0m:\u001B[0m\u001B[33m78999\u001B[0m:\u001B[0m\u001B[33m21\u001B[0m)\n"]
